packages/core/database/src/metadata/relations.ts

```diff
@@ -15,6 +16,10 @@ interface JoinTableOptions {
   meta: Meta;
 }
 
+const ID = identifiers.ID_COLUMN;
```

Comment:
I really like this pattern of storing the identifiers in constants, wonder if it could be useful to do something similar for the documentId and locale